cuebot/src/test/java/com/imageworks/spcue/test/util/FrameSetTests.java

```diff
@@ -84,4 +84,20 @@ public void shouldStopBeforeTheEndOfTheRange() {
 
         assertEquals("55-60", result.getChunk(0, 10));
     }
+
+    @Test
+    public void shouldReturnLastFrame() {
+        FrameSet result1 = new FrameSet("1-10x2");
+
+        FrameSet chunk1 = new FrameSet(result1.getChunk(1, 3));
+        FrameSet chunk2 = new FrameSet(result1.getChunk(3, 3));
+
+        assertEquals(5, chunk1.get(-1));
+        assertEquals(9, chunk2.get(-1));
+
+        FrameSet result2 = new FrameSet("1");
+        FrameSet chunk3 = new FrameSet(result2.getChunk(1, 3));
+
+        assertEquals(1, chunk3.get(-1));
```

Comment:
Same problem here with the negative index.
Also I think you meant to access the first chunk (index 0) instead of the second (index 1).
Here's a version that compiled and passed the test :)

```suggestion
        FrameSet chunk1 = new FrameSet(result1.getChunk(0, 3));
        FrameSet chunk2 = new FrameSet(result1.getChunk(3, 3));

        assertEquals(5, chunk1.get(chunk1.size()-1));
        assertEquals(9, chunk2.get(chunk2.size()-1));

        FrameSet result2 = new FrameSet("1");
        FrameSet chunk3 = new FrameSet(result2.getChunk(0, 3));

        assertEquals(1, chunk3.get(chunk3.size()-1));
```